clab/deploy_options.go

```diff
@@ -0,0 +1,79 @@
+package clab
+
+import (
+	"math"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tklauser/numcpus"
+)
+
+type DeployOptions struct {
+	reconfigure    bool
+	skipPostDeploy bool
+	graph          bool
+	maxWorkers     uint
+	exportTemplate string
+}
```

Comment:
since Setters are implemented this struct should be unexported.
To prevent users from using it directly.

If direct usage is allowed, then setters are not required

Reply:
Wait, the DeployOptions must be exported, because it will be handed in from externally.
The settings within are not, they can only be set via the setters... So in my view this is kinda right... not?!

Reply:
yeah, I didn't notice unexported fields at first